static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.tsx

```diff
@@ -624,6 +682,12 @@ const PageContentWithoutPadding = styled(PageContent)`
   padding: 0;
 `;
 
+const VisualizationWrapper = styled('div')`
```

Comment:
![image](https://user-images.githubusercontent.com/29228205/154968113-ed7c06ea-8a3d-4297-9610-ed9379adceb5.png)

to be consistent